static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -27,7 +27,7 @@ interface Props {
 
 export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
   const organization = useOrganization();
-  const url = eventData?.tags.find(tag => tag.key === 'url');
+  const url = eventData?.tags?.find(tag => tag.key === 'url');
```

Comment:
the TS type is not set to nullable (the feedback type is based on the issues `Group` type) which is weird that the `tags` is missing in the first place

Reply:
i tested here https://github.com/getsentry/sentry/pull/71123 that the feedback event should be created with an empty `tags` list even if the issue has no `tags` for some reason, so it's odd that we're missing tons of stuff in the event

Reply:
@joshacheson We no longer attach events since we'd like users to be able to submit feedback even if Sentry doesn't catch an error. So feedback is no longer backed by events, it's actually its own event type!